experiments/transfer_learn_benchmark.sh

```diff
@@ -29,7 +29,7 @@ while true; do
       REWARD_MODELS_DIR="tests/testdata/reward_models"
       NEED_TEST_FILES="true"
       SEEDS=(0)
-      extra_configs=("${extra_configs[@]}" environment.fast rl.fast train.fast fast)
+      extra_configs=("${extra_configs[@]}" environment.fast rl.fast fast)
```

Comment:
Why remove? You had been replacing them elsewhere

Reply:
Good catch! I added it back again as `policy_evaluation.fast`